test/unit/model/remove-attribute.test.ts

```diff
@@ -0,0 +1,33 @@
+import { DataTypes } from '@sequelize/core';
+import { expect } from 'chai';
+import size from 'lodash/size';
+import { getTestDialectTeaser, sequelize } from '../../support';
+
+describe(getTestDialectTeaser('Model'), () => {
```

Comment:
looks like you migrated remove-attribute.test.js to TypeScript. I'm thankful for it but don't forget to delete the existing remove-attribute.test.js test